source/server/active_stream_listener_base.cc

```diff
@@ -40,7 +43,65 @@ void ActiveStreamListenerBase::newConnection(Network::ConnectionSocketPtr&& sock
     return;
   }
   stream_info->setFilterChainName(filter_chain->name());
-  auto transport_socket = filter_chain->transportSocketFactory().createTransportSocket(nullptr);
+  bool match = false;
```

Comment:
Why don't you implement these lines of codes for matcing as a utility function?

Reply:
It is a good idea.  I will try.